src/esbmc/main.cpp

```diff
@@ -3,9 +3,12 @@
 #include <langapi/mode.h>
 
 #include <irep2/irep2.h>
-
+#include <util/config.h>
```

Comment:
Missing new line

Reply:
Newline has been added after the include.